.gitignore

```diff
@@ -1,2 +1,3 @@
 *.pyc
 *.retry
+*~
```

Comment:
Please remove this, so that the patch only has stageuser plugin code.

Reply:
ack.  This will be removed.